apps/labs/posts/integrating-hypothesis-into-sympy.md

```diff
@@ -0,0 +1,107 @@
+---
+title: 'Integrating Hypothesis into SymPy'
+authors: [diane-tc]
+published: September 20, 2023
+description: 'Gives an introduction to the utility of hypothesis in SymPy'
+category: [Property Based Testing]
+featuredImage:
+  src: /posts/integrating-hypothesis-into-sympy/blog_feature_org.png
+  alt: 'An illustration of a brown hand holding up a microphone, with some graphical elements highlighting the top of the microphone.'
+hero:
+  imageSrc: /posts/integrating-hypothesis-into-sympy/blog_hero_var2.svg
```

Comment:
Note - I don't see the image on my local website build, although everything seems right here. We need to verify if this works in the preview deployment.